test/e2e/run_test.go

```diff
@@ -500,4 +500,47 @@ var _ = Describe("Podman run", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(0))
 	})
+
+	It("podman run --volumes-from flag", func() {
```

Comment:
Can we break this into a few separate tests? We have known timeout issues in Travis, so let's try to avoid stuffing too much in any one test.

Reply:
Fixed